ocserv-authentication-ad-googleauth.md

```diff
@@ -0,0 +1,105 @@
+## How to setup ocserv for Microsoft AD Authentication and Google Authenticator as a 2FA
+
+Author: Dionis Pelivan
+
+
+### Scope
+
+This Recipe provides step by step instructions on how to install, configure,
```

Comment:
Maybe this sentence should repeat that this is for both Microsoft AD and google authenticator OATH.